pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -263,15 +259,53 @@ func (r *ReconcileIBMBlockCSI) updateStatus(instance *ibmblockcsi.IBMBlockCSI) e
 	if instance.Status.ControllerReady && instance.Status.NodeReady {
 		phase = csiv1.DriverPhaseRunning
 	} else {
+		key, value := r.getRolloutRestartAnnotation()
```

Comment:
I suggest renaming the key/value to meaningful names such as:
restartedAt, timestamp

Reply:
done